Resources/Prototypes/_Backmen/ai_factions.yml

```
@@ -207,3 +207,10 @@
   - Zombie
   - Revolutionary
 
+- type: npcFaction
+  id: XenobioSlime
+  hostile:
+  - Passive
```

yamllint (1.29.0-1)

[warning] wrong indentation: expected 4 but found 2

(indentation)